usage-data.adoc

```diff
@@ -20,6 +20,7 @@ The following table describes the opt-in usage data collected by CodeReady Conta
 |                  | Experimental features       | true/false
 |                  | tty                         | true/false
 |                  | duration                    |
+|                  | remote (Capture $SSH_TTY)   | true/false
```

Comment:
It captures if it's set or not, not the value of `$SSH_TTY`. Maybe just `remotely connected with SSH`, how it's implemented does not necessarily have to be mentioned.

Reply:
Sure, I can change the wording.

Reply:
Then maybe also de-mystify the `tty` above? Like @cfergeau says, let's just say what we find out with it (in plain English) -- whether CRC is used in a terminal or not?.

Reply:
@jsliacan not in this PR.